File: IMS-FE/src/app/core/openapi/models/purchases-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ProductDto } from '../models/product-dto';
export interface PurchasesDto {
  createdDate?: string;
  deliveryDate?: string;
  id?: number;
  productId?: number;
  products?: ProductDto;
  quantity?: number;
  supplierName?: string | null;
  supplierNumber?: string | null;
  updateDate?: string | null;
}

File: Inventory Management System/Models/Dto/PurchasesDto.cs
using Models.DB;

namespace Models.Dto
{
    public class PurchasesDto : BaseModel<long>
    {
        public string SupplierName { get; set; }
        public string SupplierNumber { get; set; }
        public int Quantity { get; set; }
        public DateTime DeliveryDate { get; set; }

        public long ProductId { get; set; }
        public ProductDto? Products { get; set; }
    }
}
